pages/classes/wise.md

```diff
@@ -99,6 +99,16 @@ You can sense and purify small amounts of [Close](../../pages/rules/distance.md)
 
 </section>
 
+<header>
+
+### Animal Form
+
+<p class="subheading">1st-level Elementalist archetype feature</p>
+
+</header>
+
+When you are unseen you can use your [Action](../../pages/combat/actions.md) to transform into an **Animal** (Common Creature, **Defence** 10 + Wisdom + Cunning, **Stamina** Wisdom + your level, **Light Melee Attack** 1d6) *or* transform back. If you lose all your **Stamina** in this form you transform back and take any left over damage.
```

Comment:
Animal Form effectively gives you Wisdom + your level temporary HP, which is equivalent to The Fiend's Dark One's Blessing feature.

Damage and Defence may be over-powered, however.